tools/StaticAnalysis/StaticAnalysis.Test/BreakingChangeAnalyzerTests.cs

```diff
@@ -600,58 +600,6 @@ public void RemoveProperty()
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
-        public void ChangeOutputGenericTypeArgument()
```

Comment:
@markcowl I need to rethink how I want to do the rules for output types, so I removed these tests for the time being until I can discuss it with you a little more